### DISCUSSION TITLE
Keycloak 22 release date

### DISCUSSION THREAD

**Discussion by KeresztesBalazs** [keycloak/keycloak#20278] [Q&A]
Hello,
I would like to know, what is the estimated release date of Keycloak version 22, as in the latest roadmap it says March/April 2023.

**Comment by sschu** [comment]
With Keycloak 21.1. released in April and a release cadence of approximately 3 months I would expect it around July/August.

**Reply by KeresztesBalazs** [reply]
Alright, thank you for your answer

**Comment by christian-2** [comment]
Duplicate of #20627.
